.vib/grafana-loki/cypress/cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```

Comment:
Needed?

Reply:
Do you refer to the whole file or just this line? 
I would say we don't need the file at all, since we are just testing the API in this case.

Reply:
The whole file.

Reply:
I'll delete it, good catch!

Reply:
Actually, I've tried running the tests without this file and they will not compile. I'll put it back. This seems to be required for Cypress to work. 
![Screenshot 2022-06-14 at 10 40 28](https://user-images.githubusercontent.com/47592680/173533848-0c7dc3fe-66bd-476d-8956-fb09573d6097.png)